LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Ability_CarbineAbilities.uc

```diff
@@ -0,0 +1,70 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    X2Ability_CarbineAbilities.uc
+//  AUTHOR:  BibiFloris
+//  PURPOSE: Adds stat modifier abilities for all tech tiers of Carbine
+//           
+//---------------------------------------------------------------------------------------
+class X2Ability_CarbineAbilities extends X2Ability
+	dependson (XComGameStateContext_Ability) config(LW_WeaponPack);
+	
+// ***** Mobility bonuses for Carbines
+var config int Carbine_CONVENTIONAL_MOBILITY_BONUS;
```

Comment:
It would be nice if you followed the conventions of using all caps for these variables